### PR TITLE
fix: bsc refuel icon

### PR DESCRIPTION
**Pull request by skylarbarrera**
Fixes APP-208

## What changed (plus any additional context for devs)
logic didnt handle BSC properly, fixed and added some extra props that should be there 

fixed the swap details refuel row as well 

## Screen recordings / screenshots
refuel sheet
bsc: https://cloud.skylarbarrera.com/Screen-Shot-2022-11-10-17-58-58.02.png

not bsc: https://cloud.skylarbarrera.com/Screen-Shot-2022-11-10-18-33-49.64.png

refuel row: 

https://cloud.skylarbarrera.com/Screen-Shot-2022-11-10-18-34-04.31.png

## What to test
peep pows

### CODE REVIEW

File: src/screens/ExplainSheet.js [hunk 1]
```diff
@@ -818,7 +818,8 @@ export const explainers = (params, colors) => ({
         ]}
       >
         <CoinIcon
-          address={params?.nativeAsset?.mainnet_address}
+          ainnet_address={params?.nativeAsset?.mainnet_address}
```
Comment (benisgold) [inline src/screens/ExplainSheet.js:821]: prop name typo

Comment (benisgold) [inline src/screens/ExplainSheet.js:821]: prop name typo

File: src/utils/ethereumUtils.ts [hunk 2]
```diff
@@ -119,8 +121,20 @@ const getNativeAssetForNetwork = async (
...
+
+    switch (network) {
+      case Network.polygon:
+        mainnetAddress = MATIC_MAINNET_ADDRESS;
+        break;
+      case Network.bsc:
+        mainnetAddress = BNB_MAINNET_ADDRESS;
+        break;
+      default:
+        ETH_ADDRESS;
```
Comment (benisgold) [inline src/utils/ethereumUtils.ts:134]: what does this line do

Comment (skylarbarrera) [reply src/utils/ethereumUtils.ts:134]: lol nothing ty

File: src/screens/ExplainSheet.js (hunk 1 above)
Comment (skylarbarrera) [reply src/screens/ExplainSheet.js:821]: 🙏

**Review by estrattonbailey — APPROVED** [review]

**Review by salman-rb — APPROVED** [review]

**Pull request merged**